azuread/configuration.go

```diff
@@ -121,7 +134,7 @@ func (p *azureFedAuthConfig) validateParameters(params map[string]string) error
 		if p.certificatePath == "" && p.clientSecret == "" {
 			return errors.New("Must provide 'password' parameter when using ActiveDirectoryApplication authentication without cert/key credentials")
 		}
-	case strings.EqualFold(fedAuthWorkflow, ActiveDirectoryDefault) || strings.EqualFold(fedAuthWorkflow, ActiveDirectoryAzCli) || strings.EqualFold(fedAuthWorkflow, ActiveDirectoryDeviceCode):
+	case strings.EqualFold(fedAuthWorkflow, ActiveDirectoryDefault) || strings.EqualFold(fedAuthWorkflow, ActiveDirectoryAzCli) || strings.EqualFold(fedAuthWorkflow, ActiveDirectoryDeviceCode) || strings.EqualFold(fedAuthWorkflow, ActiveDirectoryAzureDeveloperCli) || strings.EqualFold(fedAuthWorkflow, ActiveDirectoryEnvironment) || strings.EqualFold(fedAuthWorkflow, ActiveDirectoryWorkloadIdentity):
```

Comment:
>case strings [](http://example.com/codeflow?start=1&length=12)

is there a Go idiomatic way to convert this big statement to a map or list of values and check for containment?